### PR TITLE
chore(bd-tag): add cstor allowed bd tag annotation key

### PR DESCRIPTION
**Pull request by sonasingh46**
Signed-off-by: Ashutosh Kumar <ashutosh.kumar@mayadata.io>

### CODE REVIEW

File: pkg/apis/types/types.go [hunk 1]
```diff
@@ -121,4 +121,10 @@ const (
 	// OpenEBSCStorExistingPoolName is the name of the cstor pool already present on
 	// the disk that needs to be imported and renamed
 	OpenEBSCStorExistingPoolName = "import.cspi.cstor.openebs.io/existing-pool-name"
+
+	// OpenEBSCStorAllowedBDTagKey is the annotation key present that decides whether
+	// a particular BD with a tag is allowed in storage provisioning or not.
+	// This annotation can be used on SPC or CSPC to allow a particular BD(s) with tag
+	// for provisioning
+	OpenEBSCStorAllowedBDTagKey = "cstor.openebs.io/allowed-bd-tag"
```
Comment (akhilerm) [inline pkg/apis/types/types.go]: Is it `cstor.openebs.io/allowed-bd-tag` or `cstor.openebs.io/allowed-bd-tags`. [Here](https://github.com/openebs/cstor-operators/pull/172/files#diff-a87202891189d3ceb93dd55d219e1866R37) it is mentioned as tags

**Review by akhilerm — COMMENTED** [review]
A small typo in the annotation key. @sonasingh46 . Also I prefer to use the key from this repo in the other PR also.

**Review by mittachaitu — APPROVED** [review]
Changes are good

**Review by akhilerm — APPROVED** [review]
LGTM

**Pull request merged**